Beta v8.6.0

MichaIng opened the pull request:
### Beta v8.6.0
_(2022-06-25)_

#### New images
- Quartz64 | Support and images for the PINE64 Quartz64 series have been added to DietPi, modern feature-rich SBCs based on the new Rockchip RK3366 SoC.

#### New software
- Prometheus Node Exporter: A Prometheus exporter for hardware and OS metrics, written in Go, has been added to the DietPi software catalogue. Many thanks to @alexiri for implementing this software option: https://github.com/MichaIng/DietPi/pull/5551
- Tailscale: A new VPN server option has been added to the DietPi software catalogue. Many thanks to @goldfix for implementing this software option: https://github.com/MichaIng/DietPi/pull/5558
- Rclone: "rsync for cloud storage" has been added to the DietPi software catalogue. Many thanks to @goldfix for implementing this software option: https://github.com/MichaIng/DietPi/pull/5560
- ZeroTier: The free easy to deploy cloud-hosted VPN service has been added to the DietPi software catalogue. Many thanks to @AAS-Crypt for implementing this software option: https://github.com/MichaIng/DietPi/pull/5567

#### New features
- DietPi-Software | PiVPN: It is now possible to do an unattended install by placing a config file named "unattended_pivpn.conf" into the boot partition/directory. For example configs, have a look at https://github.com/pivpn/pivpn/tree/master/examples. Many thanks to @bastianpaetzold for implementing this feature: https://github.com/MichaIng/DietPi/pull/5546

#### Enhancements
- DietPi-Software | HAProxy: It received some updates and enhancements, using v2.6.0 sources now, enabled support for prometheus metrics, updated cipher suites, and the initial password for the admin panel is now the global software password instead of hardcoded "dietpi". All config file changes affect fresh installs only, while reinstalls will never touch the config file, but update the HAProxy binary only: "dietpi-software reinstall 98"

#### Bug fixes
- DietPi-VPN | Resolved an issue where NordVPN, IPVanish and PIA connections failed if the killswitch was enabled. Many thanks to `@jonyskids` for reporting this issue: https://dietpi.com/forum/t/dietpi-vpn-blocking-443/13348
- DietPi-LetsEncrypt | Resolved an issue where Lighttpd needed to be reloaded manually after certificate renewals to serve the new certificate. Many thanks to `@Rednox` for reporting this issue: https://dietpi.com/forum/t/letsencrypt-renew-cron-issue/13471/6
- DietPi-Software | Radarr: Resolved an issue where the installation on ARMv6 Raspberry Pi models (Raspberry Pi 1 and Zero (1)) failed, since Radarr v4 does not support Mono anymore. The latest v3 will be installed now on these models. Many thanks to @eddiermar for reporting this issue: https://github.com/MichaIng/DietPi/issues/5537
- DietPi-Software | Lidarr: Precautionary, on ARMv6 Raspberry Pi models, the latest v0.8 will be installed from now on, since the upcoming v1 won't support Mono anymore. 
- DietPi-Software | Jellyfin: Resolved an issue where the installation failed due to a missing config file. Many thanks to `@TomEighty15` for reporting this issue: https://dietpi.com/forum/t/jellyfin-installation-error/13402

MichaIng committed:
v8.5 (#5529)

- CI | Backport dependabot config for workflow updates (no actual DietPi code changes)

MichaIng committed:
Init v8.6

MichaIng committed:
v8.6

- DietPi-Build | Raise root size for Bookworm native PC images

MichaIng committed:
v8.6

- DietPi-Build | Raise root partition size for Bookworm native PC image by another 64 MiB, we're getting closer :)

MichaIng committed:
v8.6

- DietPi-Build | Initramfs generation requires even more space...

MichaIng committed:
v8.6

- DietPi-Installer | Odroid XU4 does not support rngd, even with the new kernel

MichaIng committed:
v8.6 (#5539)

- DietPi-Software | Radarr: Resolved an issue where the installation on ARMv6 Raspberry Pi models (Raspberry Pi 1 and Zero (1)) failed, since Radarr v4 does not support Mono anymore. The latest v3 will be installed now on these models. Many thanks to @eddiermar for reporting this issue: https://github.com/MichaIng/DietPi/issues/5537
- DietPi-Software | Lidarr: Precautionary, on ARMv6 Raspberry Pi models, the latest v0.8 will be installed from now on, since the upcoming v1 won't support Mono anymore.

MichaIng committed:
v8.6

- DietPi-Software | Gogs: Update fallback URL

MichaIng committed:
v8.6

- DietPi-Software | vaultwarden: Update web vault fallback URL

MichaIng committed:
v8.6

- DietPi-Software | Lidarr: Update fallback URL

MichaIng committed:
v8.6

- General | Update initramfs cleanup script

MichaIng committed:
v8.6

- Quality | Switch from for loop to more robust while-read loop over find results

MichaIng committed:
v8.6

- DietPi-Build | Quartz64 support stage 1

MichaIng committed:
v8.6

- DietPi-Installer | Quartz64 support stage 2
- DietPi-Set_hardware | Add Quartz64 1,500,000 baud rate UART support

MichaIng committed:
v8.6

- DietPi-Installer | Fix syntax

MichaIng committed:
v8.6

- DietPi-Build | Add support for multiple Quartz64 variants

MichaIng committed:
v8.6

- DietPi-Software | Gogs: Update fallback URL

bastianpaetzold committed:
v8.6 (#5546)

- DietPi-Software | PiVPN: It is now possible to do an unattended install by placing a config file named "unattended_pivpn.conf" into the boot partition/directory. For example configs, have a look at https://github.com/pivpn/pivpn/tree/master/examples. Many thanks to @bastianpaetzold for implementing this feature: https://github.com/MichaIng/DietPi/pull/5546

MichaIng committed:
v8.6

- DietPi-Software | vaultwarden: Replace full Build-Essential dependency with C compiler and headers only

Joulinar committed:
v8.6 (#5550)

- DietPi-Software | HAProxy: It received some updates and enhancements, using v2.6.0 sources now, enabled support for prometheus metrics, updated cipher suites, and the initial password for the admin panel is now the global software password instead of hardcoded "dietpi". All config file changes affect fresh installs only, while reinstalls will never touch the config file, but update the HAProxy binary only: "dietpi-software reinstall 98"

Co-authored-by: MichaIng <micha@dietpi.com>

alexiri committed:
v8.6 (#5551)

- DietPi-Software | Prometheus Node Exporter: A Prometheus exporter for hardware and OS metrics, written in Go, has been added to the DietPi software catalogue. Many thanks to @alexiri for implementing this software option: https://github.com/MichaIng/DietPi/pull/5551
- DietPi-Software | Do not apply "dietpi" user ownership if that user does not exist. We anyway want to remove it by times, so lets not strictly require it to exist.

Co-authored-by: MichaIng <micha@dietpi.com>

goldfix committed:
v8.6 (#5558)

- DietPi-Software | Tailscale: A new VPN server option has been added to the DietPi software catalogue. Many thanks to @goldfix for implementing this software option: https://github.com/MichaIng/DietPi/pull/5558
- DietPi-Software | Align IP forwarding across Tailscale, OpenVPN and WiFi Hotspot (where forced RA acceptance was missing)

Co-authored-by: MichaIng <micha@dietpi.com>

Joulinar committed:
v8.6 (#5568)

- DietPi-Software | Emby: Update fallback URL

Joulinar committed:
v8.6 (#5559)

- DietPi-Software | Jellyfin: Resolved an issue where the installation failed due to a missing config file. Many thanks to @TomEighty15 for reporting this issue: https://dietpi.com/forum/t/jellyfin-installation-error/13402

goldfix committed:
v8.6 (#5560)

- DietPi-Software | Rclone: "rsync for cloud storage" has been added to the DietPi software catalogue. Many thanks to @goldfix for implementing this software option: https://github.com/MichaIng/DietPi/pull/5560

MichaIng committed:
v8.6

- DietPi-Imager | Update clonezilla package version for older distros

AAS-Crypt committed:
v8.6 (#5567)

- DietPi-Software | ZeroTier: The free easy to deploy cloud-hosted VPN service has been added to the DietPi software catalogue. Many thanks to @AAS-Crypt for implementing this software option: https://github.com/MichaIng/DietPi/pull/5567
- DietPi-Software | Jellyfin: Avoid doubled service stop when waiting for config file and do not wait for config file if we do not touch it anyway

Co-authored-by: MichaIng <micha@dietpi.com>

MichaIng committed:
v8.6

- DietPi-Software | Fix script executable bit

MichaIng committed:
v8.6

- DietPi-Build | Add workaround for currently failing systemd-container installation on Ubuntu Jammy runner: https://github.com/actions/virtual-environments/issues/5806

MichaIng committed:
v8.6

- DietPi-Build | Fix GitHub actions runner detection

MichaIng committed:
v8.6

- DietPi-VPN | Resolved an issue where NordVPN, IPVanish and PIA connections failed if the killswitch was enabled. Many thanks to @jonyskids for reporting this issue: https://dietpi.com/forum/t/dietpi-vpn-blocking-443/13348
- DietPi-LetsEncrypt | Resolved an issue where Lighttpd needed to be reloaded manually after certificate renewals to serve the new certificate. Many thanks to @Rednox for reporting this issue: https://dietpi.com/forum/t/letsencrypt-renew-cron-issue/13471/6
- DietPi-Software | Prowlarr: Update fallback URL

MichaIng committed:
v8.6

- RC up

AAS-Crypt commented on dietpi/dietpi-software:
```diff
@@ -7552,10 +7642,10 @@ case "$1" in
+	*)
+		echo "[FAILED] Invalid command ($1), please use \"start\" or \"stop\""
+		exit 1
+	;;
```
That is questionable, can't understand what exactly was changed?

MichaIng replied:
Only replaced 8 spaces with 1 tab, to align with other lines in that script.

MichaIng committed:
v8.6

- CHANGELOG | Quartz64: Support and images for the PINE64 Quartz64 series have been added to DietPi, modern feature-rich SBCs based on the new Rockchip RK3366 SoC.

MichaIng committed:
v8.6

- DietPi-Installer | Cleanup U-Boot initramfs as well after kernel removal

MichaIng committed:
v8.6

- DietPi-Pre-patches | Create /etc/kernel/postrm.d before trying to create symlink into it
